common/technologies/air_doctrine.txt

```diff
@@ -1465,6 +1464,10 @@ technologies = {
 				ai_land_doctrine_tier_2_trigger = yes
 				factor = 5
 			}
+			modifier = {
+				factor = 0
+				tag = SCS

```

Comment:
Has this been approved by @Jas-PY?

Reply:
This is a duplicate, apparently I forgot to remove it.